### ISSUE TITLE
Installaion

### ISSUE THREAD

**Issue by Atakheir** [USEPA/Environmental-Technologies-Design-Option-Tool#4]
I try to install ETDOT_1-0 on my computer. I unzipped the files and I run setup.exe but nothing happens. I have 64bit Windows 10! Is there anyone else with the same issue.

I run it with Administrator access and I don't have any firewall to block it.

**Comment by ucchejbb** [comment]
@Atakheir Have you tried running the setup.exe in compatibility mode? I am also running 64bit Windows10 and was able to install ETDOT. I didn't have to run the setup in compatibility mode, but I can update the installation instructions if it worked for you.

**Comment by Atakheir** [comment]
Thanks, Johnathan, I tried the compatibility mode as well, but still, when I run setup.exe, it adds a task named "32-bit Setup Launcher " to my windows task manager but nothing happens after that. redownload the zip files, no luck. 

I would try it on another computer then.

**Comment by ucchejbb** [comment]
@Atakheir If you have a different computer, please try that. I recall seeing a launcher screen, but I believe that moved through fairly quickly when I installed it on my machine. Unfortunately, I don't have a second Win10 machine to try the installation on at the moment.

**Comment by Atakheir** [comment]
I tried to install it on another Win 10 64bit, and it does the same. Nothing comes up after running setup.exe.

Just to double-check, do I need to unzip "etdot_1-0.zip" and run "setup.exe"?

**Comment by ucchejbb** [comment]
@Atakheir Yes, you will likely have to fully unzip the etdot_1-0.zip file. Not just run the setup.exe from within winzip (or whatever unzipper you are using).

**Comment by Trallafitti** [comment]
I had the same problem with Windows 10 (64 bit). It turns out that the 32-bit setup launcher gets easily blocked by other programs. Analyze the wait chain in your task manager for the setup launcher and stop the processes that are shown there. [Here's a step-by-step instruction.](https://www.reddit.com/r/techsupport/comments/452lyf/32bit_setup_launcher_not_working_on_windows_10/)

**Comment by Atakheir** [comment]
@Trallafitti Thank you for the suggestion. There was a program (rundll32.exe) blocks the setup launcher. I end the rundll32.exe task and the installation continued.

I could install the ETDOT software but at the end of the installation I got this warning: 
![Untitled](https://user-images.githubusercontent.com/69496351/91342627-2af18680-e790-11ea-91ea-d1e07cffd724.png)
I pressed "okay" and the installation completed. however I couldn't run any of the software. I got different errors for different software (below). I tried different compatibility mode and I run all the softwares and the installation file with administration access.
AdOx: 
![image](https://user-images.githubusercontent.com/69496351/91343195-0053fd80-e791-11ea-80e9-08419d71234e.png)
AdDesignS:
![image](https://user-images.githubusercontent.com/69496351/91342947-a5220b00-e790-11ea-9c2e-aa281b5e291a.png)
IonExDesignS:
![image](https://user-images.githubusercontent.com/69496351/91343228-0d70ec80-e791-11ea-8e12-61f6211c4946.png)
after pressing continue:
![image](https://user-images.githubusercontent.com/69496351/91343070-d3074f80-e790-11ea-96af-68858feec44e.png)
DyeStudy, ASAP, and StEPP:
![image](https://user-images.githubusercontent.com/69496351/91343530-8cfebb80-e791-11ea-919a-d823fe058593.png)
Bioflilter:
Works
FaVOr:
Works

**Comment by ucchejbb** [comment]
@Trallafitti Thanks for that information. 
@Atakheir Trying to reinstall might work. The IonExDesignS issues was a known issue, and is not related to the installation process. The fact that some of these work is odd. And you've selected the Windows98/Me compatibility mode for the ones that don't work?

**Comment by Atakheir** [comment]
@ucchejbb I reinstalled it a couple of times, but only Bioflilter and FaVOr work properly.
To run each model, I have tried different compatibility modes, but none worked.
next I would try a Win32bit to see if it helps.

**Comment by ucchejbb** [comment]
@Atakheir - Please let me know how that goes. 

What type of Windows do you have? Home, Professional, Enterprise? Not really sure why that would matter, but just in case. Thanks.

**Comment by Atakheir** [comment]
I have Win 10 Pro 64bit.

I tried it on Windows 7 and it worked. No issue with licencing and all the modules are working properly, except "IonExds". It returned 
![image](https://user-images.githubusercontent.com/69496351/93382302-cf5a6c00-f816-11ea-808f-d234aaea74ee.png)
 You mentioned it is a known issue, do you have any solution for it?

**Comment by ucchejbb** [comment]
@Atakheir Unfortunately, I don't have a fix for the IonExDesignS issue. There are at least 2 issues in that problem, but to fix any of the modules basically requires a full update to new Visual Basic standard, which has not yet begun.

If you are interested in modeling ion exchange you may be interested in code found : https://github.com/USEPA/Water_Treatment_Models

This is not a fully developed GUI, but has code capable of predicting multi-component ion-exchange. It is a Python Based tool, so should be able to run on any computer that runs Python.
